insolver/feature_monitoring/homogeneity_report.py

```diff
@@ -0,0 +1,313 @@
+import numpy as np
+import pandas as pd
+from plotly.figure_factory import create_distplot
+from plotly import express as px
+import plotly as py
+import os
+import ntpath
```

Comment:
It would be better to use `path` or `pathlib`, but not Windows specific `ntpath` module